features/step_definitions/web_steps.rb

```diff
@@ -235,3 +235,7 @@ def with_scope(locator)
 Then /^the downloaded file should include "([^\"]*)"$/ do |content|
   page.driver.response.body.should match content
 end
+
+Then /^I wait for ajax$/ do
+  wait_for_ajax
+end
```

Comment:
This should not be an addition to this file, just an update to the existing `Then "I wait for ajax"` step definition.

Reply:
Changed.